custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -49,7 +58,17 @@ def aggregation_query(self):
                    awcm.block_name, awcm.supervisor_name, awcm.awc_name, awcm.aww_name,
                    awcm.contact_phone_number, awcm.wer_weighed, awcm.wer_eligible,
                    awcm.awc_days_open
-        )
+        );
+        /* update expected visits for cf cases (not in agg_ccs_record */
+        UPDATE {tablename}
+        SET expected_visits = expected_visits + ucr.expected
+        FROM (
+             SELECT SUM(0.39) AS expected, awc_id 
```

Comment:
What's the magic 0.39 number?

Reply:
Its the coefficient for cf cases (0.39 * num_cf_cases = cf contribution to expected visits)